src/Jobs/Feeds/Feed.php

```diff
@@ -163,39 +163,36 @@ private function generate()
             return;
         }
 
-        $expectedProductCount = $query->count();
+        $expectedProductCount = $this->getProductsCount($query);
 
         // Chunk-process the products
         $query->chunkById($this->chunkSize, function ($products) {
+            $feedLines = collect([]);
             // Map the validProducts into feed rows
-            $feedLines = $products
-                ->map(function ($product) {
+            $products
+                ->each(function ($product) use ($feedLines) {
                     try {
                         // Call the productToFeedRow method on the extending class (AdmarktFeed, BeslistFeed, etc).
                         $feedRow = $this->productToFeedRow($product);
-                        $afterCreatingFeedRowEvent = new AfterCreatingFeedRow($this->vendor, $this->store, $feedRow, $product);
-                        event($afterCreatingFeedRowEvent);
-                        $feedRow = $afterCreatingFeedRowEvent->getFeedRow();
-
-                        // Overwrite preconfigured fields for this vendor
-                        $fieldOverwrites = config('daalder-feeds.field-overwrites.'.$this->vendor);
-                        if ($fieldOverwrites) {
-                            foreach ($fieldOverwrites as $field => $value) {
-                                $feedRow[$field] = $value;
-                            }
+                        if(is_numeric(key($feedRow))) {
```

Comment:
This check can be `array_is_list()` in php 8 or `Arr::isList()` is Laravel 9.
For now you can use: `! Arr::isAssoc()` to check if it's a list.